interchaintest/fee_middleware_test.go

```diff
@@ -0,0 +1,199 @@
+package interchaintest_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	transfertypes "github.com/cosmos/ibc-go/v7/modules/apps/transfer/types"
+	relayertest "github.com/cosmos/relayer/v2/interchaintest"
+	interchaintest "github.com/strangelove-ventures/interchaintest/v7"
+	"github.com/strangelove-ventures/interchaintest/v7/chain/cosmos"
+	ibc "github.com/strangelove-ventures/interchaintest/v7/ibc"
+	"github.com/strangelove-ventures/interchaintest/v7/testreporter"
+	"github.com/strangelove-ventures/interchaintest/v7/testutil"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap/zaptest"
+)
+
+func TestFeeMiddleware(t *testing.T) {
+	if testing.Short() {
+		t.Skip()
+	}
+
+	t.Parallel()
+
+	// Get both chains
+	cf := interchaintest.NewBuiltinChainFactory(zaptest.NewLogger(t), []*interchaintest.ChainSpec{
+		{Name: "juno", ChainName: "chaina", Version: "v13.0.0", ChainConfig: ibc.ChainConfig{ChainID: "chaina", GasPrices: "0.0ujuno"}},
+		{Name: "juno", ChainName: "chainb", Version: "v13.0.0", ChainConfig: ibc.ChainConfig{ChainID: "chainb", GasPrices: "0.0ujuno"}}},
```

Comment:
If these are explicitly configured to each use 1 Validator and 0 Full Nodes then we can change the test name to something like `TestScenarioFeeMiddleware` and it will be run on CI via this command 
https://github.com/cosmos/relayer/blob/591c136dcb159601d723e87423ca3995c6bbd474/Makefile#L85

Reply:
You renamed the test but we also need to ensure that these chains are started with 0 full nodes and only 1 validator